apps/prairielearn/elements/pl-checkbox/pl-checkbox.py

```diff
@@ -450,17 +450,16 @@ def render(element_html, data):
             correct_answer_list = data["correct_answers"].get(name, [])
             if len(correct_answer_list) == 0:
                 raise ValueError("At least one option must be true.")
-            else:
-                html_params = {
-                    "answer": True,
-                    "inline": inline,
-                    "answers": correct_answer_list,
-                    "hide_letter_keys": pl.get_boolean_attrib(
-                        element, "hide-letter-keys", HIDE_LETTER_KEYS_DEFAULT
-                    ),
-                }
-                with open("pl-checkbox.mustache", encoding="utf-8") as f:
-                    html = chevron.render(f, html_params).strip()
+            html_params = {
```

Comment:
I prefer the style of this change: there shouldn't be an else after a raise in my opinion.